modules/puppet_proxy/puppet_config.rb

```diff
@@ -1,4 +1,4 @@
-require 'augeas'
+require 'augeas' if RUBY_PLATFORM =~ /ruby/
```

Comment:
I do not like it either; though removing it currently prevents smart proxy from running in windows